### PR TITLE
Removed sparkfun 404 link.

### PR DESCRIPTION
**Pull request by GrayHatter**
Two possible replacements are 
https://www.sparkfun.com/products/12576 Mate Silver 
or 
https://www.sparkfun.com/products/12580 Mate Gold
Having not seen the original link, I didn't want to replace a out dated link with something worse than nothing (expensive and non-working)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

### CODE REVIEW

**Comment by GrayHatter** [comment]
Doing some more review/investigation the link just above (Adafruit EzKey hid bluetooth board) has been discontinued as well. But the diff on github vs the one on https://beta.docs.qmk.fm/features/feature_bluetooth display the formatting differently.

**Comment by drashna** [comment]
I can't even find information on the original.  but looking at the other Mates, it looks like they are hooked up in a similar way.

But removing the line is probably best.  But ... it may be a good idea to link to sparkfun anyways. 

Such as: https://learn.sparkfun.com/tutorials/using-the-bluesmirf

**Comment by noroadsleft** [comment]
> Doing some more review/investigation the link just above (Adafruit EzKey hid bluetooth board) has been discontinued as well. But the diff on github vs the one on https://beta.docs.qmk.fm/features/feature_bluetooth display the formatting differently.

The link on Line 9 is improperly formatted, and should have been without quotation marks.

```markdown
[Adafruit EzKey HID](https://www.adafruit.com/product/1535)
```

If you could update that, that would be great IMO.

As for the Bluetooth Mate links, they should be included in the page *if* we can confirm they are viable.

```markdown
* Sparkfun [Bluetooth Mate Silver](https://www.sparkfun.com/products/12576) or [Bluetooth Mate Gold](https://www.sparkfun.com/products/12580)
```

**Comment by Rionlion100** [comment]
Any reason this has not been merged?

File: docs/feature_bluetooth.md [hunk 1]
```diff
@@ -6,14 +6,14 @@ Currently Bluetooth support is limited to AVR based chips. For Bluetooth 2.1 Qmk
...
-|[Adafruit EzKey HID]("https://www.adafruit.com/product/1535")   |Bluetooth Classic           | UART           | BLUETOOTH = AdafruitEZKey |              |
+|[Adafruit EzKey HID](https://www.adafruit.com/product/1535)   |Bluetooth Classic           | UART           | BLUETOOTH = AdafruitEZKey |              |
 |Rover Networks RN-42 (Sparkfun Bluesmirf)                       |Bluetooth Classic           | UART           | BLUETOOTH = RN42          | RN-42        |
 |[Bluefruit LE SPI Friend](https://www.adafruit.com/product/2633)|Bluetooth Low Energy        | SPI            | BLUETOOTH = AdafruitBLE   | nRF5182      |
 
 Not Supported Yet but possible:
 * [Bluefruit LE UART Friend](https://www.adafruit.com/product/2479). [Possible tmk implementation found in](https://github.com/tmk/tmk_keyboard/issues/514)
 * HC-05 boards flashed with RN-42 firmware. They apparently both use the CSR BC417 Chip. Flashing it with RN-42 firmware gives it HID capability.
-* [Sparkfun Bluetooth mate](https://www.sparkfun.com/products/14839)
+* Sparkfun Bluetooth mate
```
Comment (Rionlion100) [inline docs/feature_bluetooth.md]:
```suggestion
* Sparkfun Bluetooth Mate
```

File: docs/feature_bluetooth.md [hunk 2]
```diff
@@ -6,14 +6,14 @@ Currently Bluetooth support is limited to AVR based chips. For Bluetooth 2.1 Qmk
 
 |Board                                                           |Bluetooth Protocol          |Connection Type |Rules.mk                   |Bluetooth Chip|
 |----------------------------------------------------------------|----------------------------|----------------|---------------------------|--------------|
-|[Adafruit EzKey HID]("https://www.adafruit.com/product/1535")   |Bluetooth Classic           | UART           | BLUETOOTH = AdafruitEZKey |              |
+|[Adafruit EzKey HID](https://www.adafruit.com/product/1535)   |Bluetooth Classic           | UART           | BLUETOOTH = AdafruitEZKey |              |
 |Rover Networks RN-42 (Sparkfun Bluesmirf)                       |Bluetooth Classic           | UART           | BLUETOOTH = RN42          | RN-42        |
```
Comment (fauxpark) [inline docs/feature_bluetooth.md]:
```suggestion
|Roving Networks RN-42 (Sparkfun Bluesmirf)                      |Bluetooth Classic           | UART           |`BLUETOOTH = RN42`         | RN-42        |
```

File: docs/feature_bluetooth.md [hunk 3]
```diff
@@ -6,14 +6,14 @@ Currently Bluetooth support is limited to AVR based chips. For Bluetooth 2.1 Qmk
 
 |Board                                                           |Bluetooth Protocol          |Connection Type |Rules.mk                   |Bluetooth Chip|
 |----------------------------------------------------------------|----------------------------|----------------|---------------------------|--------------|
-|[Adafruit EzKey HID]("https://www.adafruit.com/product/1535")   |Bluetooth Classic           | UART           | BLUETOOTH = AdafruitEZKey |              |
+|[Adafruit EzKey HID](https://www.adafruit.com/product/1535)   |Bluetooth Classic           | UART           | BLUETOOTH = AdafruitEZKey |              |
```
Comment (fauxpark) [inline docs/feature_bluetooth.md]:
```suggestion
|[Adafruit EZ-Key HID](https://www.adafruit.com/product/1535)  |Bluetooth Classic           | UART           |`BLUETOOTH = AdafruitEZKey`|              |
```

File: docs/feature_bluetooth.md [hunk 4]
```diff
@@ -6,14 +6,14 @@ Currently Bluetooth support is limited to AVR based chips. For Bluetooth 2.1 Qmk
 
 |Board                                                           |Bluetooth Protocol          |Connection Type |Rules.mk                   |Bluetooth Chip|
```
Comment (fauxpark) [inline docs/feature_bluetooth.md]:
```suggestion
|Board                                                           |Bluetooth Protocol          |Connection Type |rules.mk                   |Bluetooth Chip|
```

File: docs/feature_bluetooth.md [hunk 5]
```diff
@@ -6,14 +6,14 @@ Currently Bluetooth support is limited to AVR based chips. For Bluetooth 2.1 Qmk
...
 |----------------------------------------------------------------|----------------------------|----------------|---------------------------|--------------|
-|[Adafruit EzKey HID]("https://www.adafruit.com/product/1535")   |Bluetooth Classic           | UART           | BLUETOOTH = AdafruitEZKey |              |
+|[Adafruit EzKey HID](https://www.adafruit.com/product/1535)   |Bluetooth Classic           | UART           | BLUETOOTH = AdafruitEZKey |              |
 |Rover Networks RN-42 (Sparkfun Bluesmirf)                       |Bluetooth Classic           | UART           | BLUETOOTH = RN42          | RN-42        |
 |[Bluefruit LE SPI Friend](https://www.adafruit.com/product/2633)|Bluetooth Low Energy        | SPI            | BLUETOOTH = AdafruitBLE   | nRF5182      |
```
Comment (fauxpark) [inline docs/feature_bluetooth.md]:
```suggestion
|[Bluefruit LE SPI Friend](https://www.adafruit.com/product/2633)|Bluetooth Low Energy        | SPI            |`BLUETOOTH = AdafruitBLE`  | nRF51822     |
```

**Comment by Rionlion100** [comment]
OP seems to have disappeared ill copy changes and make a new PR later

**Comment by zvecr** [comment]
Fixed by #7193.

**Pull request closed without merging**